modules/RTC/TraceablePeerConnection.js

```diff
@@ -417,33 +417,37 @@ function extractSSRCMap(desc) {
         return;
     }
 
-    session.media.forEach(function (bLine) {
-        if (!Array.isArray(bLine.ssrcs))
+    session.media.forEach(function (mLine) {
+        if (!Array.isArray(mLine.ssrcs))
         {
             return;
         }
 
-        if (typeof bLine.ssrcGroups !== 'undefined' &&
-            Array.isArray(bLine.ssrcGroups)) {
-            bLine.ssrcGroups.forEach(function (group) {
+        if (typeof mLine.ssrcGroups !== 'undefined' &&
```

Comment:
I'm renaming a var here :P can't fix all the code. This one is going away anyway.